### ISSUE TITLE
Previous watch history not importing to trakt.

### ISSUE THREAD

**Issue by xxredxpandaxx** [trakt/Plex-Trakt-Scrobbler#415]
I just set up trakt with my plex server and it properly shows what I'm currently watching and when I am finished it shows its watched. My problem is that it only added one movie that I have previously watched out of the hundreds I have marked watched, same goes for my tv shows except none were imported.

**Comment by fuzeman** [comment]
Could you enable debug logging, run a full sync (via `Plex/Web -> Channels -> Trakt.tv -> Sync`), and then post the latest plugin log file here. See [this page](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files) for more details.

**Comment by xxredxpandaxx** [comment]
Here is the log file.

[com.plexapp.plugins.trakttv.log.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/795658/com.plexapp.plugins.trakttv.log.txt)

**Comment by fuzeman** [comment]
To clarify, is this an issue with *pulling* your watched states into Plex, or *pushing* your watched states to Trakt.tv?

If watched states in Plex aren't being updated, try running a "Pull" sync (instead of "Full" or "Quick Pull"). Sometimes changes to your profile can be missed resulting in the behavior you are experiencing.

**Comment by xxredxpandaxx** [comment]
This is an issue with pushing my watched states to trakt.tv.

**Comment by xxredxpandaxx** [comment]
Any update? or is it just not going to push the previous watched content?

**Comment by fuzeman** [comment]
I couldn't see any obvious issues in your log file that indicates items are being ignored due to errors. So, in this case it might come down to:
 - Unsupported Metadata Agent *(see [this page](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/FAQ---General#3-which-metadata-agents-are-supported) for a list of supported metadata agents)*
 - You're using multiple users on your server *(only one user can be synchronized with your Trakt.tv profile currently)*
 - Other synchronization fault specific to your setup/system

Sorry for the late response, it can be difficult to keep track of all these issues..

**Comment by Ashkaan** [comment]
I'm having the identical problem running the latest version of everything.
[com.plexapp.plugins.trakttv.log](https://github.com/trakt/Plex-Trakt-Scrobbler/files/1349890/com.plexapp.plugins.trakttv.log)

**Comment by Ashkaan** [comment]
Here are DEBUG logs.  What else do you need to figure out what's going on?
[com.plexapp.plugins.trakttv.log](https://github.com/trakt/Plex-Trakt-Scrobbler/files/1351365/com.plexapp.plugins.trakttv.log)

**Comment by Ashkaan** [comment]
I just discovered that if another user watches something it marks it as watched on the main admin profile (which isn't being used for anything other than managing).  This is all messed up.  What can we do to get support here?

**Comment by rectifyer** [comment]
@Ashkaan check out https://github.com/trakt-for-plex/configuration/issues/1 for how to configure each Plex user. You would clear out any global settings (in the plugin settings screen) and instead link each user individually to their Trakt account.

**Comment by Ashkaan** [comment]
Yes, I did that.  It is set up properly to only sync 1x single Managed User to Trakt.

**Comment by rectifyer** [comment]
@Ashkaan Hmm, not really sure. I have it setup with individual users and it is working ok here. I'm using version 1.3.2 and have 2 users configured in the Plex plugin.

**Comment by Ashkaan** [comment]
How can we troubleshoot this?  I'm clearly not the only one.

**Comment by Ashkaan** [comment]
I attached logs just like you told members to do in the forums.

**Comment by Ashkaan** [comment]
I've played around with the configurator a lot and I have another issue that might be related.  I can't map User Rules to local Home accounts.

**Comment by Ashkaan** [comment]
It is sending new changes to Trakt now, but the history is not uploading.

**Comment by dossjj** [comment]
Having this same issue.
